File: exchange/Exchange.Web.BusinessLogic/Models/PhotoModel.cs
namespace Exchange.Web.BusinessLogic.Models
{
    public class PhotoModel
    {
        public string PhotoSource { get; set; }
        public string Description { get; set; }
        public long CategoryId { get; set; }
        public long? UserId { get; set; }
        public UserModel User { get; set; }
    }
}

File: exchange/exchange/exchangeClientApp/src/app/Models/photo.model.ts
export class PhotoModel {
  photoSource: string;
  description: string;
  categoryId: number;
  userId?: number;

  constructor(model: PhotoModel) {
    this.photoSource = model.photoSource;
    this.description = model.description;
    this.categoryId = model.categoryId;
    this.userId = model.userId;
  }
}
